src/res2df/grid.py

```diff
@@ -438,13 +438,12 @@ def init2df(
     usevectors = []
     include_porv = False
     for vec in init.headers:
-        if vec[1] == egrid.getNumActive() and any(
+        if vec[0] == "PORV" and any(fnmatch.fnmatch("PORV", key) for key in vectors):
+            include_porv = True
+        elif vec[1] == egrid.getNumActive() and any(
```

Comment:
Good point. 👍 This will ensure that PORV will only be process once when all cells are active.